lms/djangoapps/ccx/views.py

```diff
@@ -349,6 +349,9 @@ def visit(node, depth=1):
         widgets, which use text inputs.
         """
         for child in node.get_children():
+            # in case the children are visible to staff only, skip them
```

Comment:
Sorry, I misunderstood the socratic nature of the question. I agree about updating the docstring.